# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1291685655 28800
#      Mon Dec 06 17:34:15 2010 -0800
# Node ID b7ff4a51ea885824ef6242c34f2329592d9abe9e
# Parent  c715816f4539dfad06fb83882d66d9cf3e0ddda0
plot.m: Eliminate present tense in first sentence of docstring.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-12-06  Rik  <octave@nomad.inbox5.com>
+
+	* plot/plot.m: Eliminate present tense in first sentence of docstring.
+
 2010-12-06  Rik  <octave@nomad.inbox5.com>
 
 	* plot/xlabel.m, plot/ylabel.m, plot/zlabel.m: Add additional calling
 	form to docstring which returns handle (h)
 
 2010-12-06  Rik  <octave@nomad.inbox5.com>
 
 	* scripts/testfun/speed.m: Overhaul documentation string.
diff --git a/scripts/plot/plot.m b/scripts/plot/plot.m
--- a/scripts/plot/plot.m
+++ b/scripts/plot/plot.m
@@ -19,26 +19,28 @@
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} plot (@var{y})
 ## @deftypefnx {Function File} {} plot (@var{x}, @var{y})
 ## @deftypefnx {Function File} {} plot (@var{x}, @var{y}, @var{property}, @var{value}, @dots{})
 ## @deftypefnx {Function File} {} plot (@var{x}, @var{y}, @var{fmt})
 ## @deftypefnx {Function File} {} plot (@var{h}, @dots{})
 ## @deftypefnx {Function File} {@var{h} =} plot (@dots{})
-## Produces two-dimensional plots.  Many different combinations of
-## arguments are possible.  The simplest form is
+## Produce two-dimensional plots. 
+## 
+## Many different combinations of arguments are possible.  The simplest
+## form is
 ##
 ## @example
 ## plot (@var{y})
 ## @end example
 ##
 ## @noindent
 ## where the argument is taken as the set of @var{y} coordinates and the
-## @var{x} coordinates are taken to be the indices of the elements,
+## @var{x} coordinates are taken to be the indices of the elements
 ## starting with 1.
 ##
 ## To save a plot, in one of several image formats such as PostScript
 ## or PNG, use the @code{print} command.
 ##
 ## If more than one argument is given, they are interpreted as
 ##
 ## @example
